### ISSUE TITLE
HelixContain module secretly gives GARRISON bonus to passengers

### ISSUE THREAD

**Issue by xezon** [TheSuperHackers/GeneralsGamePatch#1873]
HelixContain module  secretly gives GARRISON bonus to passengers. TransportContain module does not. Ideally there would be an explicit INI option for `ApplyGarrisonBonusToPassengers` or similar so there is no confusion about it.

**Comment by commy2** [comment]
Yeah, a flag for this would be awesome. I think it is fine that Helix passengers get this, because you're buying a Bunker on a unit (so it should behave like one).

The flag could be used to give bonuses to Overlord bunkers to make them a bit more viable, perhaps.
